ui/components/app/transaction-list-item/transaction-list-item.component.test.js

```diff
@@ -231,10 +231,60 @@ describe('TransactionListItem', () => {
         ...(transactionGroup.primaryTransaction.custodyId = '1'),
       };
 
-      const { queryByTestId } = renderWithProvider(
+      const { queryByTestId, getByTestId } = renderWithProvider(
         <TransactionListItem transactionGroup={newTransactionGroup} />,
       );
+      const custodyIcon = getByTestId('custody-icon');
+
       expect(queryByTestId('custody-icon')).toBeInTheDocument();
+
+      expect(prettyDOM(custodyIcon)).toMatchSnapshot();
+    });
+
+    it('should display correctly the custody icon if status is signed', () => {
+      useSelector.mockImplementation(
+        generateUseSelectorRouter({
+          balance: '2AA1EFB94E0000',
+        }),
+      );
+
+      const newTransactionGroup = {
+        ...transactionGroup,
+        ...(transactionGroup.primaryTransaction.custodyId = '1'),
+        ...(transactionGroup.primaryTransaction.status =
+          TransactionStatus.signed),
+      };
+
+      const { getByTestId } = renderWithProvider(
+        <TransactionListItem transactionGroup={newTransactionGroup} />,
+      );
+
+      const custodyIcon = getByTestId('custody-icon');
+
+      expect(prettyDOM(custodyIcon)).toMatchSnapshot();
```

Comment:
It is actually not easy to see the difference between different icon doms. Is it possible to assert them by color, text?

Reply:
Fixed, thanks!

Reply:
Perfecto !